catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java

```diff
@@ -208,11 +214,31 @@ public void visit(CreatedDateMetacardUpdater createdDateMetacardUpdater) {
             }
 
             @Override
-            public void visit(SecurityClassificationMetacardUpdater securityClassificationMetacardUpdater) {
+            public void visit(
+                    SecurityClassificationMetacardUpdater securityClassificationMetacardUpdater) {
 
             }
         });
 
+        streamEndPlugin.accept(plugin -> plugin.getFactory()
```

Comment:
❓ Do we _really_ need to use the Visitor pattern for all of those? I like that pattern but it's one of the most complex ones and unless absolutely necessary we may want to use it sparingly. And chaining them make it worse.